public/src/js/utils/platform.js

```diff
@@ -0,0 +1,5 @@
+function isPlatformSpecificCollection(platform) {
+    return platform === 'App' || platform === 'Web';
```

Comment:
These strings seem to be appearing in a few places - they could be defined in `/constants/defaults.js` for clarity